spec/adapters/heroku_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+module Suspenders
+  module Adapters
+    RSpec.describe Heroku do
+      it "sets the heroku remotes" do
+        app_builder = double(app_name: app_name)
+        allow(app_builder).to receive(:append_file)
+
+        Heroku.new(app_builder).set_heroku_remotes
+
+        expect(app_builder).to have_received(:append_file).
+          with(anything, /heroku join --app #{app_name.dasherize}-production/)
```

Comment:
If we care to write these tests, why don't we care about the first argument here?